packages/client/README.md

```diff
@@ -10,100 +10,189 @@
 
 [![Build status](https://github.com/streamr-dev/monorepo/actions/workflows/client-build.yml/badge.svg)](https://github.com/streamr-dev/monorepo/actions/workflows/client-build.yml)
 ![latest npm package version](https://img.shields.io/npm/v/streamr-client?label=latest)
-[![GitHub stars](https://img.shields.io/github/stars/streamr-dev/network-monorepo?style=social)
+![GitHub stars](https://img.shields.io/github/stars/streamr-dev/network-monorepo?style=social)
 [![Discord Chat](https://img.shields.io/discord/801574432350928907.svg?label=Discord&logo=Discord&colorB=7289da)](https://discord.gg/FVtAph9cvz)
 
 This library allows you to easily interact with the [Streamr Network](https://streamr.network) from JavaScript-based environments, such as browsers and [node.js](https://nodejs.org). The library wraps a Streamr light node for publishing and subscribing to data, as well as contains convenience functions for creating and managing streams.
 
 Please see the [Streamr project docs](https://streamr.network/docs) for more detailed documentation.
 
+## Contents
+- [Important information](#important-information)
+- [Getting started](#getting-started)
+        - [Authenticating](#authenticating)
+        - [Creating a stream and publishing data to it](#creating-a-stream-and-publishing-data-to-it)
+        - [Subscribing](#subscribing)
+- [Setup](#setup)
+    - [Installation](#installation)
+    - [Importing `streamr-client`](#importing-streamr-client)
+- [Usage](#usage)
+    - [Client creation](#client-creation)
+    - [Creating a stream ](#creating-a-stream-)
+    - [Subscribing to a stream](#subscribing-to-a-stream)
+    - [Publishing to a stream](#publishing-to-a-stream)
+    - [Requesting a resend of historical events with subscriptions](#requesting-a-resend-of-historical-events-with-subscriptions)
+    - [Searching for streams](#searching-for-streams)
+    - [Interacting with the `Stream` object](#interacting-with-the-stream-object)
+        - [Getting existing streams](#getting-existing-streams)
+        - [Updating a stream](#updating-a-stream)
+        - [Stream permissions](#stream-permissions)
+        - [Deleting a stream](#deleting-a-stream)
+    - [Storage options](#storage-options)
```

Comment:
Could be simply just "Enabling storage"?

Reply:
I've changed it into "Message storage for historical data", which seems a bit more wordy, in line with previous titles. Which one should we go with?